server/integration/testsuite/pom.xml

```diff
@@ -268,6 +263,14 @@
             <artifactId>aesh</artifactId>
             <scope>test</scope>
         </dependency>
+		<dependency>
+            <groupId>javax.cache</groupId>
+            <artifactId>cache-api</artifactId>
+        </dependency>
+		<dependency>
```

Comment:
Could you please reformat it according to our formatter: https://github.com/infinispan/infinispan/tree/master/checkstyle